src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -222,12 +223,19 @@ export class CourseManagementRequests {
         });
     }
 
-    createQuizExercise(body: { course: any } | { exerciseGroup: any }, quizQuestions: [any], title = 'Cypress quiz exercise' + generateUUID(), releaseDate = day()) {
+    createQuizExercise(
```

Comment:
I would like to have some jsdoc here, which explains the parameters like the other large requests have. Make sure to mention what time unit the duration parameter uses (I assume seconds from the default value).

Reply:
Yeah that was lazy i added it now,